alica_engine/include/engine/BasicPlan.h

```diff
@@ -24,6 +24,7 @@ class BasicPlan : private RunnableObject
     using RunnableObject::getRequiresParameters;
     using RunnableObject::getTraceContext;
     using RunnableObject::getWorldModel;
+    using RunnableObject::initExecuted;
```

Comment:
Why is it necessary to make this public?

Reply:
so that it can be called from the RuleBook's transition rule